Library/Source/RuntimeImpl.cpp

```diff
@@ -28,12 +47,15 @@ namespace babylon
 
     void RuntimeImpl::UpdateSize(float width, float height)
     {
-        m_dispatcher.queue([width, height, this] { m_engine->UpdateSize(width, height); });
-    }
-
-    void RuntimeImpl::UpdateRenderTarget()
-    {
-        m_dispatcher.queue([this] { m_engine->UpdateRenderTarget(); });
+        m_dispatcher.queue([width, height, this]
+        {
+            // If this query is showing up on profiles as being noticeably inefficient,
+            // a pointer to the window could simply be cached off as runtime state. That
+            // would add a little efficiency at the cost of a little clutter, so that
+            // tradeoff should be made only if profiles show that it makes sense.
```

Comment:
I don't think you need this comment. If this is even an issue, then it should be obvious what to do.